src/Sfx/App/Scripts/Models/DataModels/Collections.ts

```diff
@@ -315,11 +318,44 @@ module Sfx {
             });
 
             this.disabledNodes = `${disabledNodes}/${disablingNodes}`;
+            this.seedNodeCount = seedNodes.length;
+
+            this.checkSeedNodeCount(this.seedNodeCount);
+            this.checkOneNodeScenario();
 
             this.healthySeedNodes = seedNodes.length.toString() + " (" +
                 Math.round(healthyNodes.length / seedNodes.length * 100).toString() + "%)";
         }
 
+        private checkSeedNodeCount(count: number) {
+            //if < 5 seed nodes display warning
+            //if count is 1, it is one box/test only scenario
+            if (count < 5 && count !== 1) {
+                this.data.warnings.addOrUpdateNotification({
+                    message: "Cluster is degraded because it does not have 5 seed nodes. See link for more details",
+                    level: StatusWarningLevel.Error,
+                    priority: 2,
+                    id: BannerWarningID.ClusterDegradedState,
+                    link: "https://aka.ms/servicefabric/durability"
+                });
+            }else {
+                this.data.warnings.removeNotificationById(BannerWarningID.ClusterDegradedState);
+            }
+        }
+
+        private checkOneNodeScenario(): void {
+            if ( !NodeCollection.checkedOneNodeScenario && this.collection.length === 1) {
+                this.data.warnings.addOrUpdateNotification({
+                    message: "One node cluster is considered test and can not perform cluster upgrades.",

```

Comment:
test => a test cluster

Seems to sound better IMO, but either ways it makes sense

can not -> cannot

Reply:
That probably will sound better.